blog/_posts/2017-05-23-whats-new-scala-native.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: blog
+post-type: blog
+by: Martin Duhem, Guillaume Massé and Denys Shabalin
+title: "What's new with Scala Native?"
+---
+
+It has been a little bit more than two months since Scala Native 0.1 has been
+released. What's new in Scala Native? What should you expect for future
+releases?
+
+## Scala Native 0.2
+
+In April, [Scala Native 0.2 was released][scala-native-0.2-release]. The main
+focus of this release was to increase the coverage of classes from
+the JDK, such that that more programs can be ported to Scala Native without any further
+effort.
+
+### Improvements to the standard library
+
+* Support for file I/O APIs from `java.io` was added by [@Duhemm][@Duhemm]
+  from the Scala Center with help from [@cedricviaccoz][@cedricviaccoz] and
+  [@Korf74][@Korf74] in [#574][#574]. Scala Native now supports enough to read
+  and write files.
```

Comment:
Could you lead into the code snippet better? E.g., "this is what it looks like now to do IO in Scala Native:"